operator/controllers/common/providers/dynatrace_test.go

```diff
@@ -133,7 +134,11 @@ func TestEvaluateQuery_CorrectHTTP(t *testing.T) {
 			TargetServer: svr.URL,
 		},
 	}
-	_, _ = kdp.EvaluateQuery(context.TODO(), obj, p)
+	r, raw, e := kdp.EvaluateQuery(context.TODO(), obj, p)
+	require.True(t, errors.IsNotFound(e))
+	require.Equal(t, []byte(nil), raw)
```

Comment:
If this works here, please use :) 
```suggestion
	require.Equal(t, nil, raw)
```

Reply:
cannot this is what prometheus returns